coralproject/talk
# [CORL-2774] fix id typo in German stream localizations

## What does this PR do?

Fixes an id typo in the German stream localizations.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- See typo in translation file
- See it is fixed
- Compare id of updated translation file to the other translation files
     - Confirms the id is correct

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

No special considerations.
